Makefile

```diff
@@ -0,0 +1,118 @@
+#   Copyright The beehive Authors.
+
+
+# Root directory of the project (absolute path).
+ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
can you use `realpath ./` instead of `dir $(abspath $(lastword $(MAKEFILE_LIST)))`?

Reply:
ok